rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -56,13 +73,28 @@
     @NonNull
     private final Sw360ReleaseService releaseService;
 
+    @NonNull
+    private final Sw360LicenseService licenseService;
+
+    @NonNull
+    private final Sw360VulnerabilityService vulnerabilityService;
+
+    @NonNull
+    private final Sw360AttachmentService attachmentService;
+
     @NonNull
     private final RestControllerHelper restControllerHelper;
 
     @RequestMapping(value = PROJECTS_URL, method = RequestMethod.GET)
-    public ResponseEntity<Resources<Resource<Project>>> getProjectsForUser(OAuth2Authentication oAuth2Authentication) {
+    public ResponseEntity<Resources<Resource<Project>>> getProjectsForUser(@RequestParam(value = "name", required = false) String name,
+    		OAuth2Authentication oAuth2Authentication) {
         User sw360User = restControllerHelper.getSw360UserFromAuthentication(oAuth2Authentication);
-        List<Project> projects = projectService.getProjectsForUser(sw360User);
```

Comment:
I would prefer to use variable names like projectFromtDatabase or sw360Projects to have a differential between the sw360 thrift objects and rest objects.

Reply:
fixed